KioskTests/Bid Fulfillment/RegistrationMobileViewControllerTests.swift

```diff
@@ -19,5 +19,20 @@ class RegistrationMobileViewControllerTests: QuickSpec {
             subject.bidDetails.newUser.phoneNumber = "1234567890"
             expect(subject).to( haveValidSnapshot() )
         }
+
+        it("unbinds bidDetails on viewWillDisappear:") {
+            let runLifecycleOfViewController = { (bidDetails: BidDetails) -> RegistrationMobileViewController in
+                let subject = RegistrationMobileViewController.instantiateFromStoryboard(fulfillmentStoryboard)
+                subject.bidDetails = bidDetails
+                subject.loadViewProgrammatically()
+                subject.viewWillDisappear(false)
```

Comment:
unless you really have to call this directly, it should probably be the `[subject beginTransition:NO animated:NO]` and `[subject endTransition]`